src/Codeception/Module/Symfony.php

```diff
@@ -532,6 +532,32 @@ public function grabService($service)
         }
         return $container->get($service);
     }
+    
+     /**
+     * @param string  $command
```

Comment:
Please add a valid description with a code example here in docblock

Reply:
fixed